### DISCUSSION TITLE
adapt explain text to toggle state

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2968] [Ideas / Feature Requests]
In for example the crowdfund app, there is the `Make Crowdfund Public` toggle. When it is turned off, the explain text of `The crowdfund is only visible to you. To make it visible to anyone else, enable this.` makes sense.

However, when the toggle is turned on, then the explain text remains the same, but it no longer makes sense. It should be changed to something more accurate like: `The crowdfund is visible to anyone. To make it only visible to you, disable this.`

![screenshot_20211012_145242](https://user-images.githubusercontent.com/30683012/136959908-56f37dea-b11a-4fce-ab55-5f73eaa05c16.png)

**Comment by bolatovumar** [comment]
Good catch @MaxHillebrand. PR to fix this here: https://github.com/btcpayserver/btcpayserver/pull/3101
